tests/integ/scala/test_dataframe_aggregate_suite.py

```diff
@@ -1336,7 +1332,7 @@ def test_listagg_within_group(session):
     )
     Utils.check_answer(
         df.group_by("color")
-        .agg(listagg("length", ",").within_group(df.length.asc()))
```

Comment:
just curious why we did this change, and are they equivalent?

Reply:
The default sorting order is ascending so I wanted to make sure that was the case with this change. These two statements are equivalent in live mode.